src/proof/nova.rs

```diff
@@ -397,11 +405,53 @@ mod tests {
         );
     }
 
+    #[test]
+    fn outer_prove_invalid_num_equal() {
+        outer_prove_aux(
+            "(= 5 nil)",
+            |store| store.nil(),
+            Status::Error,
+            3,
+            DEFAULT_CHUNK_FRAME_COUNT,
+            DEFAULT_CHECK_NOVA,
+            true,
+            100,
+            false,
+        );
+        outer_prove_aux(
+            "(= nil 5)",
+            |store| store.num(5),
+            Status::Error,
+            3,
+            DEFAULT_CHUNK_FRAME_COUNT,
+            DEFAULT_CHECK_NOVA,
+            true,
+            100,
+            false,
+        );
+    }
+
+    #[test]
+    fn outer_prove_quote_end_is_nil_error() {
+        outer_prove_aux(
+            "(quote (1) (2))",
+            |store| store.num(1),
+            Status::Error,
+            3,
+            DEFAULT_CHUNK_FRAME_COUNT, // This needs to be 1 to exercise the bug.
```

Comment:
If this needs to be 1 to exercise a known bug, it should probably be manually set to 1 here, especially since that is not the specified default.

Reply:
Sorry, pasted the comment by mistake. I will remove it.